director/extensions.py

```diff
@@ -26,15 +25,30 @@ def __init__(self):
 
     def init_app(self, app):
         self.app = app
-        self.path = Path(self.app.config["DIRECTOR_HOME"]).resolve() / "workflows.yml"
-        with open(self.path) as f:
-            self.workflows = yaml.load(f, Loader=yaml.SafeLoader)
-
+        self.load_workflows()
         self.import_user_tasks()
         self.read_schemas()
 
+    def load_workflows(self):
+        folder = Path(self.app.config["DIRECTOR_HOME"]).resolve()
+        file_fmt = self.app.config["WORKFLOW_FORMAT"]
+        if file_fmt == 'yaml':
+            self.load_yaml_workflows(folder)
+        elif file_fmt == 'json':
+            self.load_json_workflows(folder)
+        else:
+            raise ValueError(f"Invalid workflow format: '{file_fmt}'")
+
+    def load_json_workflows(self, folder):
+        with open(folder / "workflows.json") as f:
```

Comment:
How does this work? Isn't this an attempt to divide strings? 

I'd expect: 
```suggestion
        with open(f"{folder}/workflows.json") as f:
```

Reply:
This is a feature of the python pathlib library for joining paths and strings